### DISCUSSION TITLE
Use storage plugin for htaccess file

### DISCUSSION THREAD

**Discussion by angaz** [verdaccio/verdaccio#3068] [Ideas]
I would like to use the aws-s3-storage plugin, but I see that the htaccess plugin uses the `fs` module directly, so it will always write to the filesystem. It would be nice if it could also use the storage plugin for storing this file so that it can survive container restarts.

My other solution is to use a config map and force manual user creation. This is a fine solution for me. But it might not be for others. Just an idea I thought I would put out there.

**Comment by juanpicado** [comment]
Currently, plugins storage and auth are separated and you would need to create a custom auth plugin that stores in another place rather the file system.

I'm thinking about how to improve that and plugins are not tight to one single layer, so you could create plugins for different reasons. That's my goal for v6, but there is no progress at this point on plugins, I guess that would satisfy your needs.
